doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,184 @@
+
+## Configuration using a Terminal
+
+Find the IP address the gateway. This can be done in various ways. You can connect your machine to the same local network as that of the gateway Ethernet connection and scan for open SSH ports or assign a static IP to the gateway and use that. Once the gateway IP address is found, ssh into it.
+
+```bash
+$ ssh root@<GatewayIP>
+```
+
+The default username is **root**, and the default password can be also found in the sticker. It is typically suggested that you change the default password using `passwd` after logging in.
+
+Your gateway should come with a slightly modified version of the [Lora-net UDP packet forwarder](https://github.com/Lora-net/packet_forwarder) pre-installed at `/opt/nas-lgw`. There are two configuration files `global_conf.json` and `local_conf.json`, both located in `/opt/nas_lgw`.
+
+Typically, you should not need to touch the `global_conf.json` file. Instead, edit `local_conf.json`
+
+```bash
+$ cp /opt/nas-lgw/local_conf.json /opt/nas-lgw/local_conf.json.bak
+$ vi /opt/nas-lgw/local_conf.json
+```
+
+Edit the server parameters.
+
+1. **gateway_ID**: Set to the gateway EUI (in lowercase).
+2. **server_address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) this is the same as what you use instead of `https://thethings.example.com`.
+3. **serv_port_up**: UDP upstream port of the Gateway Server, typically 1700.
+4. **serv_port_down**: UDP downstream port of the Gateway Server, typically 1700.
+
+You can also update the rest of the parameters, as you see fit. For example, update the **ref_latitude**, **ref_longitude** and **ref_altitude** parameters to update the Fake GPS location reported by the gateway in its status messages.
+
+Save the configuration and restart the packet forwarder:
+
+```bash
+$ systemctl restart nas_lgw
+```
+
+If your configuration was successful, your gateway will connect to {{% tts %}} after a couple of seconds. You can easily check if your gateway has connected using the Console.
+
+The rest of the document discusses further topics that are not required for the gateway to work, but may be useful for your deployments.
+
+
+## Troubleshooting
+
+If the gateway does not connect to {{% tts %}} after a few minutes, issue a `reboot` command, or disconnect and reconnect the power supply to power-cycle the gateway.
+
+You can access the gateway system logs using journalctl. Use the `-f` flag to follow the journal (i.e. do not exit and display new logs as they arrive), `-u nas_lgw` to only show the logs related to the packet forwarder, and `-n 1000` to show the last 1000 log lines.
+
+```bash
+$ journalctl [-f] [-u nas_lgw] [-n 1000]
+```
+
+
+## Extras: VPN
```

Comment:
I think most of the things below are out of scope for this documentation (which is about connecting to The Things Stack).

@rvolosatovs @KrishnaIyer @johanstokking what do you think? Where should this content go instead?

Reply:
Yeah the extra stuff don't seem relevant here. I think the best place is [TTN docs](https://www.thethingsnetwork.org/docs/gateways/) for now.

Reply:
Removed it from the guide, keeping as template/reference. Though I am not sure that the steps would apply directly to other gateway models.